app/javascript/appliance-calculator.js

```diff
@@ -0,0 +1,12 @@
+try {
+    window.parent.postMessage(
+    {
+      id: "#appliance_calculator",
+      height: document.body.scrollHeight + 10,
+    },
+    "*"
```

Comment:
It's best practice to specify a domain here but since the data being passed to the parent window is not sensitive, it's ok to use a wildcard.